pkg/addon/type.go

```diff
@@ -57,8 +67,11 @@ type Meta struct {
 
 // DeployTo defines where the addon to deploy to
 type DeployTo struct {
-	ControlPlane   bool `json:"control_plane"`
-	RuntimeCluster bool `json:"runtime_cluster"`
+	// These two field keep the compatible for older case
+	LegacyControlPlane   bool `json:"control_plane"`
+	LegacyRuntimeCluster bool `json:"runtime_cluster"`
+	ControlPlane         bool `json:"controlPlane"`
+	RuntimeCluster       bool `json:"runtimeCluster"`
```

Comment:
Do we have a convert function? We should unifiy data from legacy, use `ControlPlane` and `RuntimeCluster` for the rest logic.

Reply:
All usage for `ControllPlane` or `RuntimeCluster` field are  in funtion like `isDeployToRuntimeOnly` and this hides the difference.